public/room.html

```diff
@@ -120,25 +122,41 @@
                 camera="userHeight: 1.6"
                 personal-space-bubble
                 look-controls
-                networked="template: #head-template; showLocalTemplate: false;" 
+                networked="template: #head-template; showLocalTemplate: false;"
             ></a-entity>
 
             <a-entity
                 id="body"
                 body-controller="eyeNeckOffset: 0 -0.11 0.09; neckHeight: 0.05"
-                networked="template: #body-template;" 
+                networked="template: #body-template;"
             ></a-entity>
-            
+
             <a-entity
                 id="nametag"
                 networked="template: #nametag-template; showLocalTemplate: false;"
             ></a-entity>
 
             <a-entity
                 id="left-hand"
-                split-axis-events
                 hand-controls2="left"
-                axis-dpad="centerZone: 1"
+                tracked-controls
+                haptic-feedback="hand:left;"
+                vive-controls-extended="
+                    hand:left;
+                    dpad_haptic_intensity: none;
+                    dpad_deadzone: 0.6;
+                    dpad_pressed_haptic_intensity: low"
+                oculus-touch-controls-extended="
+                    dpad: true;
+                    dpad_deadzone: 0.8;
+                    dpad_livezone: 0.3;
+                    dpad_directions: 8;
+                    dpad_turbo: true;
+                    dpad_haptic_intensity: none"
```

Comment:
Maybe most, if not all, of these settings should be the default, so there's less noise in the html?